tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -152,9 +155,24 @@ oe_result_t gen_report(oe_enclave_t* enclave)
                 NULL, remote_report, report_size, &parsed_report);
             if (result != OE_OK)
             {
-                log("Failed to verify report. result=%u (%s)\n",
+                printf(
+                    "Failed to verify report. result=%u (%s)\n",
                     result,
                     oe_result_str(result));
+
+                // Print TCB Info to console if verification failed
+                printf("\nQEID: ");
+                for (uint64_t n = 0; n < 16; n++)
+                {
+                    printf("%02x", quote->user_data[n]);
+                }
+                printf("\nCPU_SVN: ");
+                for (uint64_t n = 0; n < SGX_CPUSVN_SIZE; n++)
+                {
+                    printf("%02x", quote->report_body.cpusvn[n]);
+                }
+                printf("\nPCE_SVN: %02x\n\n", quote->pce_svn);
```

Comment:
nit: Prefix the value with `0x`

Reply:
Use oe_hex_string()

---
In reply to: [395719377](https://github.com/openenclave/openenclave/pull/2708#discussion_r395719377) [](ancestors = 395719377)